pkg/version/version.go

```diff
@@ -1,43 +0,0 @@
-/*
-Copyright 2017 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package version
-
-import (
-	"fmt"
-
-	"github.com/blang/semver"
-)
-
-// CRIContainerdVersion is the version of the cri plugin.
-var CRIContainerdVersion = "UNKNOWN"
-
-func validateSemver(sv string) error {
```

Comment:
does containerd already have a validateSemver unit test?

Reply:
No, they don't. I feel like this can be a CRI validation test instead. Unless kubelet stops relying on it.

Reply:
Filed an issue for this.
https://github.com/containerd/cri/issues/672